plutus-contract/test/Spec/TxConstraints/MustPayToOtherScript.hs

```diff
@@ -205,19 +260,36 @@ mkMustPayToOtherScriptPolicy t = case t of
     MustPayToOtherScript vh d v            -> Constraints.checkScriptContext @() @() (Constraints.mustPayToOtherScript vh d v)
     MustPayToOtherScriptAddress vh svh d v -> Constraints.checkScriptContext @() @() (Constraints.mustPayToOtherScriptAddress vh svh d v)
 
+{-# INLINEABLE mkMustPayToOtherScriptPolicyV2 #-}
+mkMustPayToOtherScriptPolicyV2 :: ConstraintParams -> V2.Scripts.ScriptContext -> Bool
+mkMustPayToOtherScriptPolicyV2 t = case t of
+    MustPayToOtherScript vh d v            -> V2.Constraints.checkScriptContext @() @() (Constraints.mustPayToOtherScriptInlineDatum vh d v)
+    MustPayToOtherScriptAddress vh svh d v -> V2.Constraints.checkScriptContext @() @() (Constraints.mustPayToOtherScriptAddressInlineDatum vh svh d v)
```

Comment:
@james-iohk Should we keep V1 and V2 in the same module or move the tests into separate modules (something like `Spec.TxConstraints.MustPayToOtherScript.V1.hs` and `Spec.TxConstraints.MustPayToOtherScript.V2.hs`?

Reply:
I can maybe at least provide a separate test group for V2.